src/main/java/study2/domain/Ladder.java

```diff
@@ -0,0 +1,27 @@
+package study2.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Ladder {
+
+	private List<Line> lines;
+
+	public Ladder(Players players, Height height) {
+		this.lines = makeLadder(players, height);
+	}
+
+	private List<Line> makeLadder(Players players, Height height) {
+		int playerNumbers = players.getPlayersCount();
+
+		return IntStream.rangeClosed(1, height.getHeight())
+				.mapToObj(i -> new Line(playerNumbers, false))
+				.collect(Collectors.toList());
+	}
+
+	public List<Line> getLadder() {
+		return lines;
```

Comment:
여기도 동일하게 lines의 참조를 통해 컬렉션의 값을 수정할 수 있을 것 같습니다.

불변을 보장하고 싶으시다면 Straem이나 Collections.unmodifiable을 사용해주세요!

```suggestion
		return Collections.unmodifiable(lines);
```